essential-documentation/contribute-to-appflowy/write-for-appflowy/drafts/draft-how-to-contribute-to-appflowy.md

```diff
@@ -1,2 +1,280 @@
 # \[Draft] How to contribute to AppFlowy
 
+
+<!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#introduction">Introduction</a>
+      <ul>
+        <li><a href="#general-purpose-of-this-article">General Purpose of this Article</a></li>
+      </ul>
+    </li>
+    <li>
+      <a href="#how-to-contribute-to-appflowy">How to Contribute to Appflowy</a>
+      <ul>
+        <li><a href="#how-to-connect-with-the-team">How to Connect with the Team</a></li>
+        <li><a href="#how-to-identify-and-solve-your-first-pr">How to identify and solve your first PR</a></li>
+        <li><a href="#how-to-propose-new-features-and-changes-for-appflowy">How to propose new features for Appflowy (via GitHub Issues)</a></li>
+      </ul>
+    </li>
+    <li><a href="#an-example-on-how-to-handle-your-first-pr">An Example on how to handle your first PR</a></li>
+    <li><a href="#how-to-best-engage-with-the-community">How to best engage with the community</a></li>
+    <li><a href="#links-to-resources-to-help-get-started-with-technical-contributions">Links to Resources to help get started with Technical Contributions</a></li>
+    <li><a href="#other-initiatives-by-appflowy">Other Initiatives by Appflowy</a></li>
+    <li><a href="#conclusion">Conclusion</a></li>
+  </ol>
+</details>
+
+
+
+<!-- Introduction -->
+## Introduction
+<p>Appflowy is a productivity tool that streamlines note taking and task management. It is open-source, which means that its code is available to everyone for development and customization! Whether you're an aspiring developer, a seasoned programmer, or someone genuinely interested in technology, this guide is here to help you navigate the exciting world of open-source with Appflowy!</p>
```

Comment:
```suggestion
<p>Appflowy is a productivity tool that streamlines note taking and task management. It is open-source, which means that its code is available to everyone for development and customization!</p>
```

I understand that we want to be welcoming to the readers of this article, but I'd remove this content as it is not on topic.